Kentico.Kontent.Management.Tests/Data/CodeSamples/ContentType.json

```diff
@@ -22,6 +22,17 @@
     }
   ],
   "elements": [
+    {
+      "maximum_text_length": null,
+      "name": "Tweet link",
+      "guidelines": "",
+      "is_required": true,
+      "type": "text",
+      "external_id": "20bf9ba1-28fe-203c-5920-6f9610498fb9",
+      "id": "f09fb430-2a58-59cf-be03-621e1c367501",
+      "codename": "tweet_link",
+      "is_non_localizable": true
```

Comment:
@JanBarton you only added `is_non_localizable` property to two JSON files, but in reality MAPI sets this property for all content type elements, I think we should add it to all samples and mock data files so that they correspond with the real MAPI responses. Otherwise, we can't be sure that our change doesn't break anything in the SDK.

For example, when I tried searching for `is_required` string in the solution, I found 100+ matches in 10+ files.